src/all/taddyink/build.gradle

```
@@ -0,0 +1,12 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlinx-serialization'
+
+ext {
+    extName = 'Taddy INK (Webtoons)'
```

Can it be named like the site names itself, so it's less confusing for the final users? If yes, the same package name can be kept, doesn't need to change it. If not, there's no problem into keeping as it is.

```suggestion
    extName = 'Taddy'
```